src/collector.ts

```diff
@@ -132,6 +156,15 @@ class NaiveGomodParser {
             return dependencies;
         }, []);
 
+        goModDeps = goModDeps.map(dep => {
+            const entry = replaceDependencies.get(dep.name.value);
```

Comment:
This has to be changed, as we might have more than one replace statement for the same package with different version.